recipe-server/client/control_new/components/common/ShieldTestPage.js

```diff
@@ -0,0 +1,49 @@
+import { Card, Row, Col } from 'antd';
+import autobind from 'autobind-decorator';
+import React from 'react';
+
+import ShieldIdenticon from 'control_new/components/common/ShieldIdenticon';
+
+
+@autobind
+export default class ShieldTestPage extends React.Component {
```

Comment:
I'm fine with dropping this page, I don't see a huge need for it after landing the shields. I'm open to opposing thoughts though.